helm-chart-sources/appscope/templates/service.account.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: appscope-cert-sa
```

Comment:
This is referenced as `{{ include "appscope.fullname" . }}-cert-sa` in other places.

Reply:
Done.